terraform/stacks/unimelb/data_archive/byob_ica_v2.tf

```diff
@@ -255,7 +255,8 @@ data "aws_iam_policy_document" "production_data" {
     }
     actions = [
       "s3:ListBucket",
-      "s3:GetObject"
+      "s3:GetObject",
```

Comment:
Why `GetObject`?

Reply:
`GetObject` is the existing one, Flo. I am adding new `GetObjectAttributes` (i.e. for `HeadObject` purpose). This requires modifying/adding a comma here. (see left side in diff/compare view)

Having said, this is for FM Lambda role (similarly as Portal Lambda role below). Those service Lambda use case involves reading the content.